<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="4" hostname="DESKTOP-QG8CIRA" failures="1" tests="2" name="Default test(failed)" time="0.101" errors="0" timestamp="2024-02-23T13:54:30 IST">
  <testcase classname="testNGprac.TestNGPrac" name="username" time="0.025">
    <failure type="java.lang.AssertionError" message="expected [expectedresult] but found [actualresult]">
      <![CDATA[java.lang.AssertionError: expected [expectedresult] but found [actualresult]
at testNGprac.TestNGPrac.username(TestNGPrac.java:21)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 33 stack frames]]>
    </failure>
  </testcase> <!-- username -->
  <testcase classname="testNGprac.TestNGPrac" name="password" time="0.0">
    <skipped/>
  </testcase> <!-- password -->
  <testcase classname="testNGprac.TestNGPrac" name="tearDown" time="0.0">
    <ignored/>
  </testcase> <!-- tearDown -->
  <testcase classname="testNGprac.TestNGPrac" name="launch" time="0.0">
    <ignored/>
  </testcase> <!-- launch -->
  <testcase classname="testNGprac.TestNGPrac" name="login" time="0.0">
    <ignored/>
  </testcase> <!-- login -->
  <testcase classname="testNGprac.TestNGPrac" name="nagivateToApplication" time="0.0">
    <ignored/>
  </testcase> <!-- nagivateToApplication -->
</testsuite> <!-- Default test(failed) -->
